src/Microsoft.ML.StandardTrainers/FactorizationMachine/FactorizationMachineTrainer.cs

```diff
@@ -225,7 +220,7 @@ private void Initialize(IHostEnvironment env, Options options)
             _lambdaLatent = options.LambdaLatent;
             _learningRate = options.LearningRate;
             _numIterations = options.NumberOfIterations;
-            _norm = options.Normalize;
+            _norm = (options.NormalizeFeatures == NormalizeOption.Yes);
```

Comment:
>NormalizeFeatures [](start = 29, length = 17)

Just curious, why you decide not have NormalizeOption.Auto here?

Reply:
By default normalization should be off as per Wei-Sheng

---
In reply to: [266086395](https://github.com/dotnet/machinelearning/pull/2964#discussion_r266086395) [](ancestors = 266086395)

Reply:
`public bool Normalize = true;`
so this was mistake which we fixing.
Ok, if @wschin says so.

---
In reply to: [266129940](https://github.com/dotnet/machinelearning/pull/2964#discussion_r266129940) [](ancestors = 266129940,266086395)